tools/tokenserver/test_process_account_events.py

```diff
@@ -0,0 +1,244 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file was adapted from [this file](https://github.com/mozilla-services/tokenserver/blob/master/tokenserver/tests/test_process_account_events.py) on the legacy Tokenserver